tests/sys/geom/class/geom_subr.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 # $FreeBSD$
 
+: ${TMPDIR=/tmp}
+TEST_MDS_FILE="${TMPDIR}/test_mds.$(basename $0)"
```

Comment:
This make the filename predictable, which is less safe than using `mktemp`.

Reply:
Oh, right.  I forgot that Kyua generates fresh tmpdirs.  The code should be fine either way then.